templates/js/custom-copilot-basic/README.md.tpl

```diff
@@ -0,0 +1,102 @@
+# Overview of the AI Chat Bot template
```

Comment:
Each template may need different README, do we really need this in basic template?

Reply:
This basic template is the whole template, not the common part that will be reused by other templates.
![image](https://github.com/OfficeDev/TeamsFx/assets/37978464/044917bb-9bc4-40ae-becc-317368dd09cb)